src/models/disassemblyoutput.h

```diff
@@ -17,6 +17,8 @@ struct DisassemblyOutput
     {
         quint64 addr = 0;
         QString disassembly;
+        QString linkedFunctionName;
+        int linkedFunctionOffset;
```

Comment:
= 0;

also document what this offset means, it's probably the offset into the `disassembly` string, or?

Reply:
it is the offset from the start of the function